openapi.yaml

```diff
@@ -0,0 +1,675 @@
+
+  /v2/service_instances/{instance_id}/last_operation:
+    get:
+      summary: last requested operation state for service instance
+      tags:
+        - ServiceInstances
+      operationId: serviceInstance.lastOperation.get
+      parameters:
+        - $ref: '#/components/parameters/APIVersion'
+        - name: instance_id
+          in: path
+          description: instance id of instance to find last operation applied to it
+          required: true
+          schema:
+            type: string
+        - name: service_id
+          in: query
+          description: id of the service being used
+          schema:
+            type: string
+        - name: plan_id
+          in: query
+          description: id of the plan being used
+          schema:
+            type: string
+        - name: operation
+          in: query
+          description: a provided identifier for the operation
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ServiceInstanceLastOperation'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+        '410':
+          description: Gone
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+
+  /v2/service_instances/{instance_id}/service_bindings/{binding_id}:
+    put:
+      summary: generation of a service binding
+      tags:
+        - ServiceBindings
+      operationId: serviceBinding.binding
+      parameters:
+        - $ref: '#/components/parameters/APIVersion'
+        - $ref: '#/components/parameters/OriginatingIdentity'
+        - name: instance_id
+          in: path
+          description: instance id being bound
+          required: true
+          schema:
+            type: string
+        - name: binding_id
+          in: path
+          description: binding id being bound
+          required: true
+          schema:
+            type: string
+      requestBody:
+        description: parameters for the requested service binding
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ServiceBindingRequest'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ServiceBinding'
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ServiceBinding'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
```

Comment:
I'm not 100% sure, but I think `error` and `description` field should be allowed to be returned with a 400

Reply:
I made an [issue](https://github.com/openservicebrokerapi/servicebroker/issues/403) to clear this up.